mne/annotations.py

```diff
@@ -977,3 +977,42 @@ def events_from_annotations(raw, event_id="auto",
                 (list(event_id_.keys()),))
 
     return events, event_id_
+
+
+@verbose
+def annotations_from_events(events, sfreq, first_samp=0, orig_time=None,
+                            verbose=None):
```

Comment:
you should allow to pass an event_id as a dict to have string descriptions

Reply:
i've added one. I had to write my own `_check_events` function as I  couldn't find an elegant way to use the existing ones. WDYT ?